confgenerator/testdata/valid/linux/metrics-no_conf/input.yaml

```diff
@@ -11,18 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-logging:
-  receivers:
-    syslog:
-      type: files
-      include_paths:
-      - /var/log/messages
-      - /var/log/syslog
-  exporters:
-    google:
-      type: google_cloud_logging
+metrics:
```

Comment:
Should this test case be renamed to "disable_all_metrics" instead? Same for the windows test.

Reply:
Same here. Will rename the file after the waves of GA PRs pass to avoid rebase conflicts with others.